index.html

```diff
@@ -6,24 +6,94 @@
+				<span>leave a comment</span>
+			</div>
+			<div class="articles-div">
+				<div id="article-01" class="article">
+					<img src="images/img_01.jpg" />
+					<span>lifestyle</span>
+					<h1>More then just a music festival</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sagittis ligula vel libero condimentum pharetra. Quisque sed ultricies odio. Donec imperdiet dolor in nunc pulvinar, vel placerat nibh finibus. In mattis laoreet auctor.</p>
+				</div>
+				<div id="article-02" class="article">
+					<img src="images/img_02.jpg" />
+					<span>lifestyle</span>
+					<h1>Life tastes better with coffee</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sagittis ligula vel libero condimentum pharetra. Quisque sed ultricies odio. Donec imperdiet dolor in nunc pulvinar, vel placerat nibh finibus. In mattis laoreet auctor.</p>				
+				</div>
+				<div id="article-03" class="article">
+					<img src="images/img_03.jpg" />
+					<span>photodiary</span>
+					<h1>American dream</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sagittis ligula vel libero condimentum pharetra. Quisque sed ultricies odio. Donec imperdiet dolor in nunc pulvinar, vel placerat nibh finibus. In mattis laoreet auctor.</p>				
+				</div>
+				<div id="article-04" class="article">
+					<img src="images/img_04.jpg" />
+					<span>photodiary</span>
+					<h1>A day exploring the Alps</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sagittis ligula vel libero condimentum pharetra. Quisque sed ultricies odio. Donec imperdiet dolor in nunc pulvinar, vel placerat nibh finibus. In mattis laoreet auctor.</p>				
+				</div>
+			</div>
+			<div class="newsletter-div">
+					<form action="index.html" method="post">
+					<legend>Sign up for our newsletter!</legend>
+					<input type="email" id="email" name="user_email" placeholder="Enter a valid email adress" />
+					</form>
+			</div>
+			<div class="articles-div">
+				<div id="article-05" class="article">
+					<img src="images/img_05.jpg" />
+					<span>lifestyle</span>
+					<h1>Top 10 songs for running</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sagittis ligula vel libero condimentum pharetra. Quisque sed ultricies odio. Donec imperdiet dolor in nunc pulvinar, vel placerat nibh finibus. In mattis laoreet auctor.</p>
+				</div>
+				<div id="article-06" class="article">
+					<img src="images/img_06.jpg" />
+					<span>lifestyle</span>
+					<h1>Cold winter days</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sagittis ligula vel libero condimentum pharetra. Quisque sed ultricies odio. Donec imperdiet dolor in nunc pulvinar, vel placerat nibh finibus. In mattis laoreet auctor.</p>
+				</div>
+				<div class="btn-div">

```

Comment:
Suffix `-section` niewiele zmienia :p Powinieneś móc sobie zmieniać typ elementu HTML jak masz taką potrzebę bez konieczności zmian w CSS by to kod miał sens :p `.articles-section` to może być po prostu `.articles` lub np. `articles-list`, a `.btn-div` może być np. `.articles-footer`em lub jeśli ma być tez w innych miejscach nie tylko pod artykułami to np. `.btn-wrapper`-em ;)